{"name": "test_create_order_with_auth", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 200\n +  where 403 = <Response [403]>.status_code", "trace": "self = <test_create_order.TestCreateOrder object at 0x102fc5a90>\norder_data = {'ingredients': ['61c0c5a71d1f82001bdaaa6d', '61c0c5a71d1f82001bdaaa70']}\nauth_token = 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY3NDdjZWRjOWVkMjgwMDAxYjUyNjdlNCIsImlhdCI6MTczMjc2NTcxNiwiZXhwIjoxNzMyNzY2OTE2fQ.hyg02b9ncJNjwbAmJNbF9k8d7tCbh4bgXaM0HaolCTA'\n\n    @allure.feature(\"Создание заказа\")\n    @allure.story(\"Создание заказа с авторизацией\")\n    def test_create_order_with_auth(self, order_data, auth_token):\n        headers = {\"Authorization\": f\"Bearer {auth_token}\"}\n        response = requests.post(f\"{BASE_URL}/orders\", json=order_data, headers=headers)\n>       assert response.status_code == 200\nE       assert 403 == 200\nE        +  where 403 = <Response [403]>.status_code\n\ntest_create_order.py:13: AssertionError"}, "start": 1732765716014, "stop": 1732765716178, "uuid": "66046d76-edc7-43f0-90ba-8131abbff622", "historyId": "464bb3c9d13e4175367160ee9092f7e1", "testCaseId": "464bb3c9d13e4175367160ee9092f7e1", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_with_auth", "labels": [{"name": "story", "value": "Создание заказа с авторизацией"}, {"name": "feature", "value": "Создание заказа"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "25889-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}